notebooks/image_models/solutions/object_detection_automl_vertex.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<p>It seems metrics doesn't have 'meanAveragePrecision' key when IOU is too high.</p><p><a href="https://user-images.githubusercontent.com/6895245/171187802-38be934b-e964-4339-947a-5cbd9d5f177c.png" target="_blank">This</a> is output from my model. </p><p>It returns KeyError in thee for loop since it doesn't have 'meanAveragePrecision' key when IOU is 0.95.</p><p> </p><p>So It would be safe to modify this part to:</p><pre class="ql-syntax" spellcheck="false">for metric in list(evals.metrics["boundingBoxMetrics"]):
&nbsp; &nbsp; try:
&nbsp; &nbsp; &nbsp; &nbsp; print(f"Mean Average Precision (mAP): {metric['meanAveragePrecision']}")
&nbsp; &nbsp; except KeyError:
&nbsp; &nbsp; &nbsp; &nbsp; print('IOU threshold is too high to detect data as positive.')
&nbsp; &nbsp; print(f"Intersection over union (IoU): {metric['iouThreshold']}\n")
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>